backend/src/controllers/llm.py

```
@@ -1,104 +1,86 @@
                     params.input,
                     config=config,
-                    context=self._init_context(params),
+                    context=self.streaming_service.init_context(params.model),
                 )
                 return response
         except Exception as e:
             logger.exception(f"Error in llm_invoke: {e}")
-            await self._update_store(agent, config)
+            if agent and config:
+                await self.streaming_service.update_store(agent, config)
             raise e
```

Ruff (0.14.13)

Use `raise` without specifying exception name

Remove exception name

(TRY201)